### PR TITLE
rocksdb-stats

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
- add rocksdb-stats doc. code [pr](https://github.com/vesoft-inc/nebula/pull/2243)
- update dynamically changeable rocksdb parameters. code [pr](https://github.com/vesoft-inc/nebula/pull/2291)
- comment out unsupported data type

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -1,65 +1,62 @@
...
-
-### Maintainer
+## Maintainer
 
 Maintainers are the planner and designer of the repository, with the right to merge branches into the master. The appointment is for one year. A Maintainer should:
 
 - Set technical directions, roadmap, and priorities for the repository
 - Drive the development forward and ensure newcomers, as well as long-time contributors, have a great experience
-- Ensure the overall quality of the repository
+- Make sure the overall quality of the repository
```
Comment (oldLady344) [inline community/README.md:18]: it is not appropriate to replace "ensure" with "make sure" directly here.

File: docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/job-manager.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ What's in the returned results:
 - `12/17/19 17:21:30` is the start time, which can never be empty because the initial status is running
 - `12/17/19 17:21:30` is the end time, which is empty when the job status is running. The value is set when the job status is finished, failed and stopped
 
-**Note:** There are five job status, i.e. QUEUE, RUNNING, FINISHED, FAILED, STOPPED. Status switching is described below:
+> **NOTE**: There are five job status, i.e. QUEUE, RUNNING, FINISHED, FAILED, STOPPED. Status switching is described below:
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/job-manager.md:67]: job status ==> job statuses or job states

File: docs/manual-EN/3.build-develop-and-administration/7.monitor/5.rocksdb-statistics.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+For example:
+
+```bash
+curl -L "http://172.28.2.1:12000/rocksdb_stats?stats=rocksdb.bytes.read,rocksdb.block.cache.add"
+
+rocksdb.block.cache.add=14
+rocksdb.bytes.read=1632
+```
+
+Get part of the RocksDB statistics in Json format by using the following command:
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/7.monitor/5.rocksdb-statistics.md:56]: Json => JSON

File: community/README.md (hunk 1 above)
Comment (Amber1990Zhang) [reply community/README.md:18]: Pay attention to the overall quality of the repository

File: docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/job-manager.md (hunk 2 above)
Comment (Amber1990Zhang) [reply docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/job-manager.md:67]: job states

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Review by oldLady344 — APPROVED** [review]

**Pull request merged**